test/support/tokens.js

```diff
@@ -74,10 +74,13 @@ tokens.standardIdTokenClaims = {
   'auth_time': 1449696330
 };
 
+tokens.now = 1449699929000;
```

Comment:
Maybe we could use the Factory pattern (as we are doing for idx responses) to generate tokens via parameters, instead of having them implicitly share values like "time".

Reply:
I was trying to do some refactor like that in a separate PR (to keep the current change small), probably it would be good to do it now.